src/Compilers/Core/Rebuild/VisualBasicCompilationFactory.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.VisualBasic;
+using Microsoft.Extensions.Logging;
+using Roslyn.Utilities;
+using CS = Microsoft.CodeAnalysis.CSharp;
+using VB = Microsoft.CodeAnalysis.VisualBasic;
+
+namespace BuildValidator
+{
+    public class VisualBasicCompilationFactory : CompilationFactory
+    {
+        public new VisualBasicCompilationOptions CompilationOptions { get; }
```

Comment:
Could we implement this as `(VisualBasicCompilationOptions)base.ParseOptions`? Similar with CompilationOptions below.

Reply:
Also, the order of these two members is not consistent between C#, VB, and the base type. It would be nice to clean that up.

Reply:
Fixed along with the similar suggestion for C#